packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "@justeat/f-cookie-banner",
   "description": "Fozzie Cookie Banner - Cookie Banner",
-  "version": "4.5.0",
+  "version": "4.6.0",
   "main": "dist/f-cookie-banner.umd.min.js",
-  "maxBundleSize": "40kB",
+  "maxBundleSize": "80kB",
```

Comment:
It might look like we've increased the bundle size with this PR, however previously we just weren't building static cookie banner as part of CI. 👍🏼